optimum/onnxruntime/modeling_ort.py

```diff
@@ -206,7 +210,7 @@ def _from_pretrained(
             )
             kwargs["model_save_dir"] = Path(model_cache_path).parent
             kwargs["latest_model_name"] = Path(model_cache_path).name
-            model = ORTModel.load_model(model_cache_path)
+            model = ORTModel.load_model(model_cache_path, provider=provider)
```

Comment:
lets define in the `load_model` a `self.provider` attribute easily access the provider of the model.

Reply:
I don't know if we'd be able to access an attribute since this `load_model` call is static. Unsure what the request is here

Reply:
we can extract the provider from the model and then set it/ provider it in the `cls` call. There should be an easy way for a user to know about the provider and device used for the model, e.g. `model.device`; `model.provider`.